cpp/include/cudf/io_types.h

```diff
@@ -23,27 +23,47 @@
  */
 #pragma once
 
+typedef enum 
+{
+  FILE_PATH,
+  BUFFER,
+  URL
+} resource_type;
+
+typedef struct 
+{  
+  char			*data;			/**< pointer to a host memory buffer	*/
```

Comment:
Please ensure comments align.

Reply:
Also data members should align. You are using tabs, which causes them not to align on github (or in anyone's editor who uses spaces).

Reply:
Same here. The alignment is consistent with the rest of the file, which uses tabs and assumes 4 spaces per tab.